### ISSUE TITLE
Question: How i can show a react native screen when touching bubble?

### ISSUE THREAD

**Issue by longb1997** [HarryDoan/ReactNative_Android_NativeModule#1]
Hello, as the title says, can I use my react native screen when I touch the bubble chat? And how I can do it?
Thank you for the great library

**Comment by DustinDoan315** [comment]
Certainly! You can navigate to the home screen by tapping the bubble on the outside. If you want to navigate to another screen, such as the detail screen or any other, you can manage the navigation logic directly from the home screen.
